pubspec.yaml

```diff
@@ -11,35 +11,25 @@ dependencies:
   flutter:
     sdk: flutter
   cupertino_icons: ^1.0.2
-  cln_grpc:
-    git:
-      url: https://github.com/dart-lightning/lndart.cln_grpc.git
-  cln_common:
-    git:
-      url: https://github.com/dart-lightning/lndart.clightning.git
-      path: packages/cln_common
-  clightning_rpc:
-    git:
-      url: https://github.com/dart-lightning/lndart.clightning.git
-      path: packages/rpc
+  cln_grpc: ^0.0.1-beta.2
+  cln_common: ^0.0.1-beta.1
+  clightning_rpc: 0.0.2-beta.1
   trash_component:
     git:
       url: https://github.com/vincenzopalazzo/trash
       path: packages/components
```

Comment:
we can move also this in the pubdev version? we release the version with your change, right?

Reply:
ooh super cool